faq.md

```diff
@@ -7,13 +7,29 @@ css_id: faq
 
 ### **What makes Uptane different from other SOTA security mechanisms?**
 
+Security problems can occur due to accidental disclosures, malicious attacks, or disgruntled insiders.  It is not
+a matter of whether a successful attack will occur, but when.  Because of the very real threat of a compromise, a security system must be able to securely recover from an attack. This means that an update system must have a way to restore its operations in a timely fashion when a key is lost or compromised.  
```

Comment:
```suggestion
Security problems can occur due to accidental disclosures, malicious attacks, or disgruntled insiders.  It is not a matter of whether a successful attack will occur, but when.  Because of the very real threat of a compromise, a security system must be able to securely recover from an attack. This means that an update system must have a way to restore its operations in a timely fashion when a key is lost or compromised.  
```